doc/source/serve/api/index.md

```diff
@@ -355,7 +355,6 @@ Content-Type: application/json
 .. autosummary::
    :nosignatures:
    :toctree: doc/
-   :template: autosummary/class_with_inherited_members.rst
```

Comment:
should this be deleted then?

Reply:
I think it was only added for this purpose

Reply:
oh, good catch, removed.